src/app/entity/PassableEntityConstructor.ts

```diff
@@ -0,0 +1,3 @@
+import {Entity} from './entity';
+
+export type PassableEntityConstructor<T extends Entity> = new(id: string) => T;
```

Comment:
Good idea to define a proper type for this rather than always having to search for complicated type definition!

I would simply name it `EntityConstructor` though - we don't have a non-passable one anyway.

As this is just a one liner and closely related, maybe we add it to the entity.ts file rather than in a separate one?